search-commons/src/main/java/no/unit/nva/search2/model/ParameterKey.java

```diff
@@ -39,11 +41,11 @@ enum ParamKind {
 
     enum Operator {
         NONE("%s%s"),
-        EQUALS("%s:%s"),
-        GREATER_THAN("%s:>%s"),
-        GREATER_THAN_OR_EQUAL_TO("%s:>=%s"),
-        LESS_THAN("%s:<%s"),
-        LESS_THAN_OR_EQUAL_TO("%s:<=%s"),
+        EQUALS("(%s:%s)"),
```

Comment:
why these ( ) ?

Reply:
I was suprised to see this in my code here. Thought it was in a earlier attempt to use lucene, and that it didn't go to production.
This is not in use at the moment, can change it back.